### PR TITLE
Removed old Ruby version dependencies

### PR DESCRIPTION
**Pull request by info-rchitect**

### CODE REVIEW

**Comment by pderouen** [comment]
Looks like the plugin's Gemfile needs update too

**Comment by ginty** [comment]
I don't think we should remove the byebug gem, just the version constraint

**Comment by priyavadan** [comment]
Also, Gemfile has an error (end)

Might as well remove ruby 2.2 from the travis config yml file.

**Comment by info-rchitect** [comment]
> I don't think we should remove the byebug gem, just the version constraint

OK i am fine with that but why would we tie to version 8 vs the latest?

**Comment by ginty** [comment]
I don't think we should tie to a version, should just list the name without a version in the new Gemfile

File: templates/app_generators/plugin/Gemfile [hunk 1]
```diff
@@ -12,9 +12,8 @@ source 'https://rubygems.org'
 # This gem provides integration with https://coveralls.io/ to monitor
 # your application's test coverage
 gem 'coveralls', require: false
+gem 'byebug'
```
Comment (ginty) [inline templates/app_generators/plugin/Gemfile:15]: Should not be within this external audience block

**Review by ginty — CHANGES_REQUESTED** [review]

**Comment by priyavadan** [comment]
I want to try out the change to byebug internally on our systems, before this gets merged.

**Review by ginty — APPROVED** [review]
LGTM

**Comment by priyavadan** [comment]
LGTM!

**Comment by info-rchitect** [comment]
> LGTM!

Going to merge and then do another PR for moving to github actions

**Pull request merged**